### PR TITLE
Replace RabbitMQ with SQS

### PR DESCRIPTION
**Pull request by deshraj**
This PR involves the following changes:

- [x] Replace RabbitMQ with SQS
- [x] Add ElasticMQ to mock SQS on the local machine
- [x] Update the docker settings
- [x] Add ElasticMQ in dev docker
- [x] Fix the seed script
- [x] Update README and Docs with the updated installation instructions
- [x] Remove RabbitMQ dependency in CI and add ElasticMQ in CI
- [x] Handle the case of gracefully killing the submission worker (see https://stackoverflow.com/questions/18499497/how-to-process-sigterm-signal-gracefully?answertab=active#tab-top)
- [x] Modify tests to work with the new codebase

### CODE REVIEW

**Comment by deshraj** [comment]
@RishabhJain2018: Can you please test this on your local machine? 

@isht3: you should try this on your machine and try to understand it since this will be a part of your next sprint.

File: requirements/worker.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+numpy==1.14.5
+networkx==2.1
```
Comment (RishabhJain2018) [inline requirements/worker.txt]: @deshraj This must on line 1 😛

File: scripts/workers/submission_worker.py [hunk 2]
```diff
@@ -299,6 +286,7 @@ def run_submission(challenge_id, challenge_phase, submission, user_annotation_fi
     submission.save()
     try:
         successful_submission_flag = True
+        print(EVALUATION_SCRIPTS)
```
Comment (RishabhJain2018) [inline scripts/workers/submission_worker.py]: We should replace it with logger statements.

**Comment by RishabhJain2018** [comment]
@isht3 No, you don't need the credentials for this. :)

**Comment by isht3** [comment]
Also, I added Logger statements to wherever necessary. :+1:

**Pull request merged**